addons/zeusFactory/functions/fnc_getTransportType.sqf

```diff
@@ -21,16 +21,5 @@ case (east): {"potato_e_" + _crewType};
 case (resistance): {"potato_i_" + _crewType};
 };
 
-private _maxCargoRoom = 99;
-private _addGunner = false;
-
-if (_vehType != "") then {
-    _maxCargoRoom = ([_vehType, true] call BIS_fnc_crewCount) - 1;
-    if (_transportType in [TRANSPORT_APC_RTB, TRANSPORT_APC_FOLLOW]) then {
-        _addGunner = true;
-        _maxCargoRoom = _maxCargoRoom - 1;
-    };
-};
-
 // Return:
-[_vehType, _crewType, _maxCargoRoom, _addGunner]
+[_vehType, _crewType]
```

Comment:
3rd arg of function still used here
`([_factoryLogic,_transportType,_side] call FUNC(getTransportType)) params ["", "", "_maxCargoRoom"];`
https://github.com/BourbonWarfare/POTATO/blob/7d37de240765fea41bf31f2bda6bab3b67d07a2e/addons/zeusFactory/functions/fnc_createGroup.sqf#L10